tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -34,16 +33,18 @@ def device_name(self):
 
     def enable_bluetooth(self):
         self.marionette.find_element(*self._bluetooth_label_locator).tap()
```

Comment:
Why remove the part where it is checked that the bluetooth checkbox gets in a checked state?